dev_tools/requirements/deps/runtime.txt

```diff
@@ -24,6 +24,10 @@ dash
 # quimb simulation
 quimb
 
+#qsharp resource estimator interop
```

Comment:
@tanujkhattar should we start thinking about developing external integerations like this as extensions with their own dependencies like what we do in Cirq?

Reply:
Eventually, yes. I think we can still continue to push on the development without introducing the overhead of meta packages like Cirq. But this would be a good 20% project for sure.